### DISCUSSION TITLE
[roslib] Property 'data' does not exist on type 'Message'

### DISCUSSION THREAD

**Discussion by overplex** [DefinitelyTyped/DefinitelyTyped#63916] [Issues with a @types package]
### Working code example:
![code](https://user-images.githubusercontent.com/61654129/212479405-ce1a3324-e81b-4a1b-8f93-aa587a801c6e.jpg)

### Console output:
![Console output](https://user-images.githubusercontent.com/61654129/212479279-e8cd17f1-0170-4f73-8ee1-9211f5be8289.jpg)

### Add ".data":
![Issue](https://user-images.githubusercontent.com/61654129/212479292-8b2de6ee-aac4-4036-b20e-79a930643dfa.jpg)

[a comment by an automated account is not shown]

**Comment by alumag** [comment]
Work as intended.
You are supposed to extend the `ROSLIB.Message` class to fit your needs.

Source code: https://github.com/RobotWebTools/roslibjs/blob/develop/src/core/Message.js
